src/global.css

```diff
@@ -0,0 +1,16 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
```

Comment:
You also could've added properties `font-size` and `font-weight` in html tag for default customization of all text on the page, and restyle only those, which need it. It will save a dozen lines in your .css file